### ISSUE TITLE
Improve README.md for linux build

### ISSUE THREAD

**Issue by FredLePiv** [mozilla-mobile/mozilla-vpn-client#3730]
2 errors occured by applying the command "cmake --build build -j$(nproc)"

mozilla-vpn-client/src/crashreporter/crashuploader.cpp:107:34: error: ‘sslErrors’ is not a member of ‘QNetworkReply’
mozilla-vpn-client/src/crashreporter/crashuploader.cpp:109:15: error: cannot determine the type of 'err'

Any suggestion

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-2395)

**Comment by strseb** [comment]
What Qt-Version are you using? :)

**Comment by FredLePiv** [comment]
Qt6.3.1

**Comment by birdsarah** [comment]
@FredLePiv Qt 6.2.x is required.

We definitely need to improve the README for building linux though so I've updated your ticket so we can action this. I hope that's ok. 

Things that I needed to do to build.

I had got Qt from Qt Maintenance Tool. I then needed to install a bunch of additional packages with apt to successfully build and run without runtime errors. Things like `libqt6core5compat6-dev libqt6networkauth6-dev libqt6opengl6-dev`. I could see the list I needed here: `linux/debian/control.qt6`.

Finally I had a problem with dbus and I needed to update the install command `sudo cmake --install build --prefix /usr`
